packages/playground/_stories/main/Dialog/Dialog.stories.ts

```diff
@@ -1,212 +1,135 @@
-  initial-focus="${ifDefined(args.initialFocus)}"
-  ?prevent-focus-restore="${ifDefined(args.preventFocusRestore)}"
-  ?open-by="${ifDefined(args.open)}"
-  accessible-name="${ifDefined(args.accessibleName)}"
-  accessible-name-ref="${ifDefined(args.accessibleNameRef)}"
+	id="dialog"
+	header-text="${ifDefined(args.headerText)}"
+	?stretch="${ifDefined(args.stretch)}"
+	?draggable="${ifDefined(args.draggable)}"
+	?resizable="${ifDefined(args.resizable)}"
+	state="${ifDefined(args.state)}"
+	initial-focus="${ifDefined(args.initialFocus)}"
+	?prevent-focus-restore="${ifDefined(args.preventFocusRestore)}"
+	?open="${ifDefined(args.open)}"
+	accessible-name="${ifDefined(args.accessibleName)}"
+	accessible-name-ref="${ifDefined(args.accessibleNameRef)}"
+	accessible-role="${ifDefined(args.accessibleRole)}"
 >
-  ${unsafeHTML(args.header)}
-  ${unsafeHTML(args.default)}
-  ${unsafeHTML(args.footer)}
+	${unsafeHTML(args.header)}
+	${unsafeHTML(args.default)}
+	${unsafeHTML(args.footer)}
 </ui5-dialog>
 
 <script>
-  const dialogOpener = document.getElementById("dialogOpener-${id}");
-  const dialog = document.getElementById("dialog-${id}"); 
-  const dialogCloser = document.querySelector("#dialog-${id} .dialogCloser");
-  dialogOpener.accessibilityAttributes = {
-    hasPopup: "dialog",
-    controls: dialog.id,
-  };
-  dialogOpener.addEventListener("click", function () {
-    dialog.show();
-  });
-  dialogCloser.addEventListener("click", function () {
-    dialog.close();
-  });
+	var dialogOpener = document.getElementById("dialogOpener");
+	var dialog = document.getElementById("dialog"); 
+	var dialogCloser = document.querySelector("#dialog .dialogCloser");
+
+	dialogOpener.accessibilityAttributes = {
+		hasPopup: "dialog",
+		controls: dialog.id,
+	};
+	dialogOpener.addEventListener("click", function () {
+		dialog.open = true;
+	});
+	dialogCloser.addEventListener("click", function () {
+		dialog.open = false;
+	});
 </script>`;
 };
 
 export const Basic = Template.bind({});
-
 Basic.args = {
-  headerText: "Register Form",
-  default: `
-    <section class="login-form">
-      <div>
-        <ui5-label for="username" required="">Username: </ui5-label>
-        <ui5-input id="username"></ui5-input>
-      </div>
-      <div>
-        <ui5-label for="password" required="">Password: </ui5-label>
-        <ui5-input id="password" type="Password" value-state="Error"></ui5-input>
-      </div>
-      <div>
-        <ui5-label for="email" type="Email" required="">Email: </ui5-label>
-        <ui5-input id="email"></ui5-input>
-      </div>
-      <div>
-        <ui5-label for="address">Address: </ui5-label>
-        <ui5-input id="address"></ui5-input>
-      </div>
-    </section>`,
-  footer: `
-    <div slot="footer" style="display: flex; align-items: center;padding: .5rem">
-      <div style="flex: 1;"></div>
-      <ui5-button id="dialogCloser" design="Emphasized">Register</ui5-button>
-    </div>`,
+	headerText: "Register Form",
+	default: `
```

Comment:
"default": and "footer" slots appear with extra indentation in the storybook for all samples